recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ test:
     - if not exist %PREFIX%\\Library\\lib\\gg.lib exit 1                    # [win]
     - if not exist %PREFIX%\\Library\\bin\\gg.dll exit 1                    # [win]
     # Inspect linkage
-    - ldd -v $PREFIX/lib/libgg$SHLIB_EXT                                    # [linux]
+    - ldd -v $PREFIX/lib/libgg$SHLIB_EXT                                    # [linux and build_platform == target_platform]
```

Comment:
We have emulation on Linux. Did this break something?

Reply:
line 929 of https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1002411&view=logs&jobId=6494c7a6-a019-59a3-6f3d-9bab8ca96717&j=6494c7a6-a019-59a3-6f3d-9bab8ca96717&t=18c91bca-a25a-5f89-34d4-93bbec4021bc "not a dynamic" something. 

ctests in build.sh (on other feedstocks) have been working ok with the azure cross-compile + emulation for aarch64 and ppc64le. but I've usually had to except the `ldd` meta.yaml test sections in other feedstocks, too. https://github.com/conda-forge/libxc-feedstock/blob/main/recipe/meta.yaml#L98

Reply:
Got it. Thanks!!!